documentation/release-notes.md

```diff
@@ -0,0 +1,187 @@
+# casual release notes
+
+## 1.7
+
+We're proud to announce the release of casual middleware version `1.7`.
+
+### runtime configuration 
+
+`casual` can now update all parts of configuration while running. No part is 
+restarted.
+
+A convenient way to update the configuration is using runtime edit.
+
+```console
+$ casual configuration --edit
+```
+
+see [configuration.operation.md](../middleware/administration/documentation/cli/configuration.operation.md)
+
+### service timeout
+
+The service timeout semantics have changed a bit.
+
+When a service is called that has a timeout, a deadline is set based on the timeout.
+This deadline is propagated downstream to consecutively called services.
+If a service is invoked with an ongoing deadline, the earliest of the ongoing and 
+the timeout based deadline is set for the service. 
+
+Therefore, the first service sets an _execution window_ to which downstream service 
+invocations will conform.
+
+One, perhaps negative, consequence of this semantic is that services downstream 
```

Comment:
We are using strictly objective terms and then lists a possible negative consequence, shouldn't we balance this off with something positive that explain why we did this change? :-)

Reply:
Check. Do a _suggestion_ how you want it?